infoschema/tables.go

```diff
@@ -2309,6 +2324,30 @@ func dataForTableTiFlashReplica(schemas []*model.DBInfo) [][]types.Datum {
 	return rows
 }
 
+// dataForTableTiFlashReplica constructs data for table tiflash replica info.
+func dataForMetricTables(ctx sessionctx.Context) [][]types.Datum {
+	var rows [][]types.Datum
+	tables := make([]string, 0, len(MetricTableMap))
+	for name := range MetricTableMap {
+		tables = append(tables, name)
+	}
+	sort.Strings(tables)
+	for _, name := range tables {
+		schema := MetricTableMap[name]
+		genPromQL := schema.GenPromQL(ctx, nil, schema.Quantile)
```

Comment:
I think it's unnecessary to add the `GENERATE_PROMQL` column.

Reply:
great,done.